app/src/main/java/org/gnucash/android/ui/settings/BackupPreferenceFragment.java

```diff
@@ -273,6 +276,7 @@ private void toggleGoogleDriveSync(){
 		if (appFolderId != null){
 			sharedPreferences.edit().remove(getString(R.string.key_google_drive_app_folder_id)).commit(); //commit (not apply) because we need it to be saved *now*
 			mGoogleApiClient.disconnect();
+			onGoogleDriveFolderForgot(appFolderId);
```

Comment:
Intention to notify listener(s) that the folder is no longer available and that they should do something about it.